server/config/settings.py

```diff
@@ -107,18 +109,16 @@
 
 # Internationalization
 # https://docs.djangoproject.com/en/1.9/topics/i18n/
-
-LANGUAGE_CODE = 'en-us'
-
-TIME_ZONE = 'UTC'
-
+LANGUAGE_CODE = 'he'
+LANGUAGES = [
+  ('he', _('Hebrew')),
+  ('en', _('English')),
+]
+TIME_ZONE = 'Asia/Jerusalem'
 USE_I18N = True
-
 USE_L10N = True
-
 USE_TZ = True
 
```

Comment:
This is not a must, [check this one](https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#how-django-discovers-translations)